packages/core/src/modules/discover-features/DiscoverFeaturesModuleConfig.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * DiscoverFeaturesModuleConfigOptions defines the interface for the options of the DiscoverFeaturesModuleConfig class.
+ * This can contain optional parameters that have default values in the config class itself.
+ */
+export interface DiscoverFeaturesModuleConfigOptions {
+  /**
+   * Whether to automatically accept feature queries. Applies to all protocol versions.
+   *
+   * @default true
+   */
+  autoAcceptQueries?: boolean
+}
+
+export class DiscoverFeaturesModuleConfig {
+  private options: DiscoverFeaturesModuleConfigOptions
+
+  public constructor(options?: DiscoverFeaturesModuleConfigOptions) {
+    this.options = options ?? {}
+  }
+
+  public get autoAcceptDiscoverFeatureQueries() {
```

Comment:
Might be good to add an inheritDoc here (as used in other modules) so you get documentation when getting a property from the config